.storybook/preview.js

```diff
@@ -1,22 +1,14 @@
 import React from 'react';
+// import { ThemeProvider, createTheme } from '@mui/material';
+import { muiTheme } from 'storybook-addon-material-ui5';
 
-// import { muiTheme } from 'storybook-addon-material-ui'
+// Create your own theme like this.
+// Note: you can specify theme name in `themeName` field. Otherwise it will be displayed by the number.
+// you can specify only required fields overriding the `Light Base Theme`
 
-// const newTheme = { themeName: 'Grey Theme', palette: { primary: '#00bcd4', alternateTextColor: '#4a4a4a', canvasColor: '#616161', textColor: '#bdbdbd', secondaryTextColor: 'rgba(255, 255, 255, 0.54)', disabledColor: '#757575', accent1Color: '#607d8b', }, };
-
-// export const decorators = [muiTheme([newTheme])];
-
-// import { GlobalStyles } from '../src/config/global.styles';
-
-/*
- * Global decorator to apply the styles to all stories
- * Read more about them at:
- * https://storybook.js.org/docs/react/writing-stories/decorators#global-decorators
- */
 export const decorators = [
   (Story) => (
     <>
-      {/* <GlobalStyles /> */}
       <Story />
     </>
   ),
```

Comment:
Change the decorator to use storybook-addon-material-ui5 and display the theme as explained [here](https://storybook.js.org/addons/storybook-addon-material-ui5):
```
export const decorators = [
  muiTheme()
];
```